### DISCUSSION TITLE
I have the  error _items.nut #4753 bug and cant open the console to fix it

### DISCUSSION THREAD

**Discussion by ThatOneOlive** [R2Northstar/Northstar#441] [Tech Support]
So i was installing mods on northstar client and at some point when i tried to edit my pilot i got the said error which until now has been unable me to customize my pilot, so i found out that there was a command that you can input on the console that fixes that problem and it indeed did fix it but after fixing the issue me being  dumb has hell installed another mod that gave me the same issue but when  i went for the second time to fix the problem i couldnt open the console on the main menu so i tried to rebind the key in game and on the files but no result and now im stuck and unable to open the console to fix this problem i even reinstalled the game but still nothing.

So i am here to ask if anyone every got this problem and how they fixed it. (Also if you dont know the error im talking about it is an error that when you try to edit the pilot loadout the game crashes sending you to the main menu saing the message i refered to).

Ps: I can open the game console in the middle of a game but not on the main menu

**Comment by GeckoEidechse** [comment]
Have you already tried disabling all mods except the `Northstar.XYZ` ones?

**Reply by ThatOneOlive** [reply]
Well i reinstalled everything so i should have no mods

**Reply by GeckoEidechse** [reply]
Depends on how you reinstall. Uninstalling Titanfall2 via Steam for example does not delete mods. Neither does reinstalling Northstar by dragging/dropping in a new release zip file. Only manually deleting the `R2Norhtstar\mods` folder will actually remove the mods...

**Reply by ThatOneOlive** [reply]
Oh ok then i will try it when i am able too and then i will give feedback

**Reply by F1F7Y** [reply]
`Northstar.*` Mods come with the Northstar client and implement most functionality.
Disabling them shouldn't have effect on your issue. 

Unsure if this'll work, but try adding `+ns_reset_persistance` to your startup args. This should reset your loadouts every time you start up the game so make sure you remove it afterwards.

As far as I'm aware resseting your loadouts is the only way to fix this issue.

**Reply by JMM889901** [reply]
Maybe you could post your log? If the pilot screen caused it maybe you installed/uninstalled moreskins or another mod that modifies persistence, if so might be possible to fix it without resetting persistence

**Reply by ThatOneOlive** [reply]
Oh yeah i remeber someone saying that but when i wrote there i forgot to add the + sigh at the beginning so i guess thats why it didnt work. Ty i will try it when i can and give feeback

**Reply by ThatOneOlive** [reply]
> Maybe you could post your log? If the pilot screen caused it maybe you installed/uninstalled moreskins or another mod that modifies persistence, if so might be possible to fix it without resetting persistence 

Yeah it was the moreskins mod that started it but i'll first the solution that involves the args

**Reply by ASpoonPlaysGames** [reply]
> Unsure if this'll work, but try adding `+ns_reset_persistance` to your startup args. This should reset your loadouts every time you start up the game so make sure you remove it afterwards.

its `+ns_resetpersistance` or `+ns_resetpersistence` i forget which

**Reply by F1F7Y** [reply]
Just checked launcher, it's `+ns_resetpersistence`

**Reply by ThatOneOlive** [reply]
Ok. Just a thing do i write it on both folders or is there a specific one.

**Reply by ThatOneOlive** [reply]
It worked!!!!! Thank you so much for everyones help :)

Btw which mods can cause this issue? Are there specific ones for example the ones that change the skins or any mod can do this?

**Reply by JMM889901** [reply]
Anything that messes with persistent data can, so new camos/skins etc, especially if you set to a value added by a mod then disable that mod

**Reply by ThatOneOlive** [reply]
ok ty

**Reply by ThatOneOlive** [reply]
may i close this discussion? Or should i leave it for someone who may encounter the same problem?

**Reply by crimsonneuron** [reply]
So I ran into the same problem, used +ns_resetpersistence, fixed it, and now my account and weapons don't have any levels. Is that fixable, or should I just live with it?

**Reply by F1F7Y** [reply]
That's exactly what `ns_resetpersistence` does. It resets all your loadouts and levels.

Once you fixed the error this issue is about you can remove `ns_resetpersistence`

**Reply by crimsonneuron** [reply]
I did, and still all my progress is reset. Is that just something I have to manually fix?

**Reply by uniboi** [reply]
yes

**Reply by crimsonneuron** [reply]
alright, thank you

**Reply by crimsonneuron** [reply]
Alright, trying to get my progress back. I found the command to set my player level back to what is was, but is there any command that allows me to set my prosceen number/my weapon gen? Couldn't find anything like it on northstar discord, but I've seen people with impossible proscreen numbers.

**Reply by crimsonneuron** [reply]
never mind, found it
